portfolio-project-react/src/components/WorkExpItem/WorkExpItem.jsx

```diff
@@ -0,0 +1,161 @@
+import React, { useEffect, useState } from "react";
+import styles from "./WorkExpItem.module.css";
+import deleteIcon from "./assets/ic_delete.png";
+import editIcon from "./assets/ic_edit.png";
+import {
+  LABEL_CURR_EMP,
+  LABEL_END_DATE,
+  LABEL_START_DATE,
+} from "../../constants";
+import moment from "moment";
+
+const WorkExpItem = ({ isEdit, data, index, onDelete, onUpdate }) => {
+  const [editExp, setEditExp] = useState(false);
```

Comment:
use proper names for variables and files. please follow this elsewhere as well where proper names are not used.